web/modules/custom/joinup_search/config/install/search_api.index.published.yml

```diff
@@ -415,13 +416,41 @@ processor_settings:
       - field_isr_description
       - field_ar_abstract
       - field_is_description
+      - is_field_ci_name
+      - is_field_owner_name
       - field_user_first_name
       - field_user_family_name
       - field_user_organisation
+    spaces: ''
+    overlap_cjk: 1
+    minimum_word_size: '3'
     weights:
-      preprocess_index: -50
-      preprocess_query: -50
-  rendered_item: {  }
+      preprocess_index: -49
+      preprocess_query: -49
+  type_boost:
+    boosts:
+      'entity:node':
+        datasource_boost: !!float 1
+        bundle_boosts:
+          custom_page: !!float 5
+          discussion: !!float 2
+          document: !!float 0
+          event: !!float 3
+          news: !!float 4
```

Comment:
While this boost value works correctly it can me misleading when you go to the admin UI, at `/admin/config/search/search-api/index/published/processors`:
<img width="602" alt="boost" src="https://user-images.githubusercontent.com/473868/30262943-ba47de28-96db-11e7-9a5b-d7ae2fd4203f.png">

This happens because the entered values are not in the standard list of the select dropdown, which is: 0.0,
 0.1, 0.2, 0.3, 0.5, 0.8, 1.0, 2.0, 3.0, 5.0, 8.0, 13.0, 21.0. If somebody will save that form the types with non-standard values will be set to NULL.

I would suggest that we use values from the standard list for `node:news`, `rdf_entity:collection` and `rdf_entity:solution`.

- `node:custom_page`: 8.0.
- `node:news`: 5.0.
- `node:event`: 3.0.
- `node:discussion`: 2.0.

Also "Default boost for items from this datasource" (`datasource_boost`) should be 0.0, according to the ticked.